source/actions3/BioXAS/BioXASMAXvMotorMoveToLimitActionInfo.cpp

```diff
@@ -0,0 +1,67 @@
+#include "BioXASMAXvMotorMoveToLimitActionInfo.h"
+
+BioXASMAXvMotorMoveToLimitActionInfo::BioXASMAXvMotorMoveToLimitActionInfo(const AMControlInfo &motor, CLSMAXvMotor::Limit limitSetpoint, QObject *parent) :
+	AMActionInfo3(QString(), QString(), QString(), parent)
```

Comment:
Just to be pedantic, do you want to have real values for these names and descriptions?